packages/internal-test-helpers/lib/run.ts

```diff
@@ -13,7 +13,7 @@ export function runDestroy(toDestroy: any): void {
   }
 }
 
-export function runTask(callback: (...args: any[]) => any): void {
+export function runTask<F extends () => any>(callback: F): ReturnType<F> {
```

Comment:
Was dropping the params intentional? If not:

```suggestion
export function runTask<F extends (...args: any[]) => any>(callback: F): ReturnType<F> {
```

Reply:
It isn't ever called with args so didn't seem terrible to simplify.

Reply:
Seems good!